src/status_im/ui/screens/offline_messaging_settings/events.cljs

```diff
@@ -3,18 +3,29 @@
             [status-im.utils.handlers :as handlers]
             [status-im.utils.handlers-macro :as handlers-macro]
             [status-im.ui.screens.accounts.models :as accounts.models]
+            [status-im.models.mailserver :as models.mailserver]
+            [status-im.transport.inbox :as inbox]
             [status-im.i18n :as i18n]
             status-im.ui.screens.offline-messaging-settings.edit-mailserver.events
             [status-im.utils.ethereum.core :as ethereum]))
 
 (handlers/register-handler-fx
  ::save-wnode
```

Comment:
`::save-wnode` has a side effect that should be registered with reg-fx. It would be better named `wnode/disconnect` and take the cb as a parameter.
You don't need the handlers-nacro/merge-fx macro here because you only have one effect producing function there and no chaining.
Should be something like this
```
{:wnode/disconnect [address #(re-frame/dispatch [:wnode/update chain wnode])]}
```